docs/releases/1.9.txt

```diff
@@ -65,6 +65,15 @@ the included auth forms for your project, you could set, for example::
 
 See :ref:`password-validation` for more details.
 
+Permission mixins for class-based views
```

Comment:
I think this is a major feature - I think we should also give credit to braces here which has been the de-facto way of doing this for 3 years.